src/Bicep.Core/Syntax/ResourceDeclarationSyntax.cs

```diff
@@ -81,13 +81,32 @@ public TypeSymbol GetDeclaredType(IBinder binder, IResourceTypeProvider resource
             ResourceTypeReference? typeReference;
             var ancestors = binder.GetAllAncestors<ResourceDeclarationSyntax>(this);
             if (ancestors.Length == 0)
-            {
+            {                
```

Comment:
```suggestion
            {
```

[nit] Trailing whitespaces.